FrameworklessServerTests/ControllerTests.cs

```diff
@@ -84,9 +84,31 @@ public void DeleteUserShouldReturnNotFoundResponseIfUserNameIsNull()
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
             ResetList();
         }
-    }
 
-    public class TestUserService
-    {
+        [Fact]
+        public void PutMethodShouldReturnOkStatusIfUserSuccessfullyUpdated()
+        {
+            var controller = new Controller(_usersService);
+            var bob = new User("Bob");
+            _usersService.Add(bob);
+            Assert.Contains(_usersService.GetAllUsers(), u => u.Name == bob.Name);
```

Comment:
If we add our testUserService here, we could have bob in the list so we dont have to setup service related things in this test